### PR TITLE
Fix dictionary declaration to litteral

### PR DESCRIPTION
**Pull request by remyleone**

### CODE REVIEW

File: pandas/_version.py [hunk 1]
```diff
@@ -205,10 +205,7 @@ def git_pieces_from_vcs(tag_prefix, root, verbose, run_command=run_command):
         raise NotThisMethod("'git rev-parse' failed")
     full_out = full_out.strip()
 
-    pieces = {}
-    pieces["long"] = full_out
-    pieces["short"] = full_out[:7]  # maybe improved later
-    pieces["error"] = None
+    pieces = {"long": full_out, "short": full_out[:7], "error": None}
```
Comment (jreback) [inline pandas/_version.py]: you are using a pretty old version of master as this no longer exists. pls rebase on master

**Comment by sieben** [comment]
@jreback Done.

**Comment by jreback** [comment]
``pandas/_version.py`` still is being changed. did you push?

**Comment by sieben** [comment]
I just did.

File: pandas/_version.py (hunk 1 above)
Comment (sieben) [inline pandas/_version.py]: I've just rebased on master...

Comment (jreback) [inline pandas/_version.py]: did u push -f ?

Comment (sieben) [inline pandas/_version.py]: Indeed.

2015-11-14 13:12 GMT+01:00 Jeff Reback <notifications@github.com>:

Comment (jreback) [inline pandas/_version.py]: doesn't appear to be the case. you still have the original commits.

Comment (sieben) [inline pandas/_version.py]: I've done:
git fetch upstream (with upstream being github:pydata/pandas)
git rebase upstream/master
git push -u origin -f dict_litteral:dict_litteral

Plus I've checked my code against the master and it maches. Would it help
if I submitted the modification on the _version.py in a separate pull
request?
If that's too much hassle I can close this PR...

2015-11-14 16:24 GMT+01:00 Jeff Reback <notifications@github.com>:

Comment (jreback) [inline pandas/_version.py]: try

``git push sieben dict_litteral -f``

you can't push directly to origin, only to your fork

Comment (sieben) [inline pandas/_version.py]: I've did it. Is there a portion of the work that can be merged? I don't
mind closing it...

Comment (jreback) [inline pandas/_version.py]: changes will be reflected here / it appears the same

**Comment by jreback** [comment]
pls checkout current master and rebase. pls reopen if you are updating.

**Pull request closed without merging**